### DISCUSSION TITLE
Why was support for the IBM PS/1 2133 removed?

### DISCUSSION THREAD

**Discussion by ausdev16** [86Box/86Box#2625] [General]
Can anyone advise why this was removed? I just setup an install specifically for this and it's gone!

I can't see any discussion on the commit:

Panagiotis committed on May 2, 2021 
1 parent 6b2e544 commit aae4163a8e7dc1d39088944680c720324d7d19c5

**Comment by TC1995** [comment]
It's due to the 129 error on post, which can't be fixed right now due to no documentation how to counter it.

**Reply by OlivierDoriath** [reply]
The manual mentions that this is related to cache banks and jumpers 16 and 17. Is this not implemented, or is there another issue?
